content/docs/3_reference/6_system/1_options/0_markdown/reference-article.txt

```diff
@@ -37,3 +37,17 @@ return [
   ]
 ];
 ```
+
+## Safe mode: `markdown.safe`
+
+Enables/disables Parsedown's safe mode which applies sanitisation to scripting vectors (such as scripting link destinations) that are introduced by the markdown syntax itself.
```

Comment:
From the description as an uninitiated user I wouldn't understand this. Maybe explain with an example what exactly this does?